articles/azure/azs-how-to-obtain-clientid-via-api.md

```diff
@@ -51,8 +51,10 @@ In order to obtain Access Token you will need to create a request body and use R
 
 The Authentication/Login Endpoint can be https:/\/login.microsoftonline.com/{tenant id}/oauth2/token or https:/\/login.windows.net/{tenant id}/oauth2/token
 
-> [!IMPORTANT] The  {tenant id}  value in the path of the request can be used to control who can sign into the application. The allowed values are tenant identifiers, for example,  8eaef023-2b34-4da1-9baa-8bc8c9d6a490  or  contoso.onmicrosoft.com  or  common  for tenant-independent tokens
+> [!IMPORTANT]
+> The  {tenant id}  value in the path of the request can be used to control who can sign into the application.
```

Comment:
Change this to be active rather than passive voice:
Use the {tenant id} value in the path of the request to control who can sign in to the application.